docs/sites.yml

```diff
@@ -3166,3 +3166,14 @@
   built_by: Datawire
   built_by_url: https://www.datawire.io
   featured: false
+  - title: Asian Art Collection
```

Comment:
Hi @karthikmadupu,

Thanks for showcasing the Princeton Art Museum website. Please can you correct the indentation of title here. Thank you !

Reply:
Dear Kiran,
As suggested corrections have been made.
Thanks.